js/bybit.js

```diff
@@ -5290,8 +5816,13 @@ module.exports = class bybit extends Exchange {
             throw new ArgumentsRequired (this.id + ' fetchMyTrades with inverse subType requires settle to not be USDT or USDC');
         }
         const [ type, query ] = this.handleMarketTypeAndParams ('fetchMyTrades', market, params);
-        const enableUnifiedMargin = await this.isUnifiedMarginEnabled ();
-        if (type === 'spot') {
+        const [ enableUnifiedMargin, enableUnifiedAccount ] = await this.isUnifiedEnabled ();
+        if (enableUnifiedAccount && !isInverse) {
+            if (params['orderId'] === undefined) {
```

Comment:
@sc0Vu you can't access it directly because it leads to a key error in python and PHP
![image](https://user-images.githubusercontent.com/43336371/220965760-de11acda-76ab-43ee-9dbd-5fd8c3dc5683.png)